client/types/types.go

```diff
@@ -53,3 +56,47 @@ type SecurityTest struct {
 	Default          bool          `bson:"default" json:"default"`
 	TimeOutInSeconds int           `bson:"timeOutSeconds" json:"timeOutSeconds"`
 }
+
+// HuskyCIVulnerability is the struct that stores vulnerability information.
+type HuskyCIVulnerability struct {
+	Language       string `json:"language"`
```

Comment:
@rafaveira3, would you mind adding the `omitempty` tag here as well?

Reply:
Yeah, sure! Thanks for catching this one, @Krlier. Did it at 09f9656c0cec057bcb7db6243a40e0cd44b3d302.